apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionSummary.tsx

```diff
@@ -0,0 +1,72 @@
+import { useMemo } from 'react'
+import { Flex } from 'prepo-ui'
+import { observer } from 'mobx-react-lite'
+import styled from 'styled-components'
+import { useRootStore } from '../../../context/RootStoreProvider'
+import SummaryRecord from '../../../components/SummaryRecord'
+import { EstimateYourProfitLoss, EstimatedValuation } from '../../definitions/index'
+import { compactNumber } from '../../../utils/number-utils'
+
+const Profit = styled.span`
+  color: ${({ theme }): string => theme.color.success};
+`
+
+const Loss = styled.span`
+  color: ${({ theme }): string => theme.color.error};
+`
+
+const ClosePositionSummary: React.FC = () => {
+  const { tradeStore } = useRootStore()
+  const {
+    closePositionValue,
+    closePositionValueBN,
+    selectedPosition,
+    closePositionValuation,
+    closePositionPnlAmount,
+  } = tradeStore
+
+  const pnlText = useMemo(() => {
+    // this line is only to get pass type check
+    // the returned value doesn't matter because
+    // SummaryRecord will show loading skeleton when closePositionPnlAmount is undefined
+    if (closePositionPnlAmount === undefined) return ''
+
+    // TODO: show "+ <$0.01" or "- <$0.01" if profit/loss is too small, pending actual design
+    if (compactNumber(closePositionPnlAmount) === '0') return '$0'
+
+    if (closePositionPnlAmount > 0)
+      return <Profit>+${compactNumber(closePositionPnlAmount)}</Profit>
+
+    // the replace() is to remove '-' in compacted value so the output doesn't look '$-1.23'
+    return <Loss>-${compactNumber(closePositionPnlAmount)}</Loss>
+  }, [closePositionPnlAmount])
+
+  if (!selectedPosition || closePositionValue === '' || closePositionValueBN?.eq(0)) return null
+
+  const loadingPnl =
+    closePositionPnlAmount === undefined || selectedPosition.totalValueBN === undefined
+
+  return (
+    <Flex width="100%" flexDirection="column" px={12} pb={8} gap={4}>
+      <SummaryRecord
+        label="Estimated Valuation Price"
```

Comment:
```suggestion
        label="Estimated Price"
```
same for open position